### ISSUE TITLE
Customer receives email with original customer name despite customer change

### ISSUE THREAD

**Issue by lucollab** [freescout-helpdesk/freescout#3149]
Hello all,

if I change the email address of the customer in an open ticket, the first and last name (display name) of the previous customer remains, even if the email changes.

So if I have a ticket open from the customer "Ben Wight" with the email address "[ben.wight@mail.com](mailto:ben.wight@mail.com)" in which "Mona Wight" is in CC with "[mona.red@mail.com](mailto:mona.red@mail.com)" and I want to send a reply in the ticket to "Mona Red", I click on the reply button and select "[mona.red@mail.com](mailto:mona.red@mail.com)". In FreeScout this all looks neat and correct.

In the mail client of "Mona Red", however, an e-mail comes in with the following recipient: "Ben Wight [mona.red@mail.com](mailto:mona.red@mail.com)".

Is this a deliberate behaviour or a bug?

Many thanks for your help!

Best regards
Lukas

**Comment by freescout-helpdesk** [comment]
We'll double check.

**Comment by freescout-helpdesk** [comment]
> I have a ticket open from the customer "Ben Wight" with the email address "[ben.wight@mail.com](mailto:ben.wight@mail.com)" in which "Mona Wight" is in CC with "[mona.red@mail.com](mailto:mona.red@mail.com)"

How was this ticket created?

> I click on the reply button and select "[mona.red@mail.com](mailto:mona.red@mail.com)"

If a customer simply adds mona.red@mail.com to CC when sending an email to FreeScout there will be no way to send a reply directly to mona.red@mail.com from FreeScout. Something is missing in your explanation.

**Comment by lucollab** [comment]
I have also just tried to reproduce the facts. It seems to be due to the mail client Thunderbird, in Outlook everything is displayed correctly. I am closing this ticket as it is not a freescout problem.

**Comment by avenjamin** [comment]
@lucollab maybe check the address book or recent contacts in Thunderbook to see if Mona's email address has some how become associated with Ben's contact card.

**Comment by patthewebrat** [comment]
Hi there,

Reviewing the source and based on our testing it does look like this is a FreeScout issue rather than client side. Could we re-open and have another look?

Thanks
Pat

**Comment by patthewebrat** [comment]
I've traced the root cause. In `app/Jobs/SendReplyToCustomer.php` line 280:

```php
$to = [['name' => $this->customer->getFullName(), 'email' => $this->customer_email]];
```

`$this->customer` is always the conversation's current requester, but `$this->customer_email` comes from `$this->conversation->customer_email` — which may belong to a different customer after a requester change. The wrong name is sent in the SMTP `To` header itself, not a client-side display issue. Outlook appeared to work because it overrides the header name with its own contact cache.

I've opened a PR with a fix: #5198
